src/components/Main/MainView/MessageInput/use/wordSuggester.ts

```diff
@@ -0,0 +1,176 @@
+import store from '@/store'
+import createTree, { TrieNode } from '@/lib/trieTree'
+import { animeEffectSet, sizeEffectSet } from '@traptitech/traq-markdown-it'
+import { ComputedRef, onBeforeMount, WritableComputedRef, ref } from 'vue'
+import getCaretPosition from '@/lib/caretPosition'
+import { EntityEventMap, entityMitt } from '@/store/entities/mitt'
+
+export type Target = {
+  word: string
+  begin: number
+  end: number
+  divided: boolean
+}
+
+const getCurrentWord = (elm: HTMLTextAreaElement, text: string): Target => {
+  text = text.replaceAll('　', ' ')
+  const startIndex = elm.selectionStart
+  const prevAtMarkIndex = text.lastIndexOf('@', startIndex - 1)
+  const prevColonIndex = text.lastIndexOf(':', startIndex - 1)
+  const prevPeriodIndex = text.lastIndexOf('.', startIndex - 1)
+  const nearest = Math.max(prevAtMarkIndex, prevColonIndex, prevPeriodIndex)
+  const begin = nearest < 0 ? 0 : nearest
```

Comment:
```suggestion
  const begin = Math.max(nearest, 0)
```
上で`Math.max`使ってるなら統一したさ